backend/main/strategies/staked_token_weighted_default.go

```diff
@@ -0,0 +1,63 @@
+package strategies
+
+import (
+	"fmt"
+	"math"
+
+	"github.com/brudfyi/flow-voting-tool/main/models"
```

Comment:
should this be importing from DapperCollectives/CAST instead?

Reply:
actually no in the go.mod file the package name is `module github.com/brudfyi/flow-voting-tool` so I can only import from this directory within the project. However, it's a good thing to raise @jacksonConrad, should we change the go.mod package directory?